packages/eccodes/package.py

```diff
@@ -16,7 +16,8 @@ class Eccodes(CMakePackage):
     list_url = 'https://software.ecmwf.int/wiki/display/ECC/Releases'
 
     maintainers = ['skosukhin']
-    
+
+    version('2.19.0', sha256='c4e6f453639281d2d0f3eaf32140bb4a0d8cbeb01045b73a5f2a6ca3c1fec69d')
```

Comment:
should we set to a1d080aed1b17a9d4e3aecccc5a328c057830cd4d54f451f5498b80b24c46404 according to ther error message in jenkins ?